include/cuco/detail/static_map/functors.cuh

```diff
@@ -17,17 +17,51 @@
 
 #include <cuco/detail/bitwise_compare.cuh>
 
+#include <thrust/tuple.h>
```

Comment:
Can we use `cuda::std::tuple` instead?

Reply:
It doesn't work since the map `retrieve_all` passes a zip iterator to the `open_addressing_impl` and `thrust::tuple` is not assignable from a `cuda::std::tuple`